testsuite/rhsso/objects.py

```diff
@@ -42,23 +42,26 @@ def create_user(self, username, password, **kwargs):
         self.admin.update_user(user_id, {"emailVerified": True})
         return user_id
 
-    def oidc_client(self, client_id, client_secret) -> KeycloakOpenID:
+    def oidc_client(self, client_id, client_secret, cert=None) -> KeycloakOpenID:
         """Create OIDC client for this realm"""
         return KeycloakOpenID(
             server_url=self.admin.connection.server_url,
             client_id=client_id,
             realm_name=self.name,
             client_secret_key=client_secret,
+            cert=cert,
+            verify=False,
```

Comment:
This might need to be configurable in the future, if the SSO will be using same cert as cluster, it might make sense to verify.